ui/pages/token-details/index.scss

```diff
@@ -0,0 +1,50 @@
+.token-details {
+  width: 359px;
```

Comment:
Where does this number come from?

Reply:
@darkwing This number come from Figma design: https://www.figma.com/file/EUQ1LPxKPU9tga4YnjISn1/Network-specific-assets?node-id=634%3A12028
Here is screenshots about that property:
![Screenshot 2022-02-07 at 13 14 21](https://user-images.githubusercontent.com/92527393/152797021-7cfa9698-9563-4f8b-a35d-00cca2d40358.png)
![Screenshot 2022-02-07 at 13 14 57](https://user-images.githubusercontent.com/92527393/152797041-05b28b50-6592-4d10-9628-445d3de29de3.png)

P.S. In Figma you can see design only for modal view, but I made the page display on expand view as well.

Reply:
You are right @brad-decker, in this case the popup window is 359 px wide.
I have small fix for expanded view here: 6432d927a47e25e8774fe137b0de015ca531bf7f
Now `Token Details` page is looking and displays good as @segun said. 
Here is screenshot in expanded view:

![Screenshot 2022-02-11 at 10 48 05](https://user-images.githubusercontent.com/92527393/153573206-9c6fed29-339a-4d5d-bc49-54dcf90dfdf8.png)